core/services/fs/src/core.rs

```diff
@@ -205,4 +216,61 @@ impl FsCore {
             .map_err(new_std_io_error)?;
         Ok(())
     }
+
+    /// Get user metadata from extended attributes on Unix systems.
+    ///
+    /// OpenDAL stores user metadata in xattr with the prefix "user.opendal.".
```

Comment:
Please don't add our own prefix. We should be able to read xattr set by other applications.